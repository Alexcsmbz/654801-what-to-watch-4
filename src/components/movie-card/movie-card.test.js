import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const movie = {
  name: `Film 1`,
  img: `img/img-1.jpg`,
};

describe(`MovieCard snapshot test`, () => {
  it(`MovieCard should render movie`, () => {
    const tree = renderer.create(
        <MovieCard
          movie={movie}
          onMouseEnter={() => {}}
          onMouseLeave={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
