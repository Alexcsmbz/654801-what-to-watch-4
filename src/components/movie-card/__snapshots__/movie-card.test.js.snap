// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard snapshot test MovieCard should render movie 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <video
    className="movie-player"
    controls={true}
    height="175"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    width="280"
  >
    <source
      type="video/webm"
    />
    <source
      type="video/mp4"
    />
  </video>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/movie-page/film-1"
      onClick={[Function]}
    >
      Film 1
    </a>
  </h3>
</article>
`;
